opensource/cli/src/commands/application.ts

```diff
@@ -17,6 +17,17 @@ async function listAllApplicationsHandler() {
   prettyPrint(apps);
 }
 
+async function dbDumpHandler() {
+  const dump = await fetchCordManagementApi<ApplicationData[]>(
```

Comment:
This seems like it should be `fetchCordManagementApi<string>`

Reply:
true I'll change it